types/block.go

```diff
@@ -13,6 +13,15 @@ import (
 	cmn "github.com/tendermint/tendermint/libs/common"
 )
 
+const (
+	// MaxHeaderBytes is a maximum header size (including amino overhead).
+	MaxHeaderBytes = 478
+
+	// MaxAminoOverheadForBlock - maximum amino overhead to encode a block (up to
+	// MaxBlockSizeBytes in size).
+	MaxAminoOverheadForBlock = 4
```

Comment:
verify

Reply:
I know. Should be 4 + 1 byte for each field (4) probably

> We'd have to calculate a maximum number of txs to create the MaxAminoOverheadForBlock

no. see what I've done with MaxAminoOverheadForTx. In short: when we add txs, we count len(tx)+MaxAminoOverheadForTx

Reply:
It's still seems a bit complex. If you or anybody else have ideas how to simplify this stuff, I'd be happy to hear them